src/components/Slider.js

```diff
@@ -1,137 +1,161 @@
-import React, { useMemo } from "react";
+import React from "react";
 import Slider from "react-slick";
-import { useLocation } from 'react-router-dom';
 
 const ArrowLeft = () => {
-  return (
-    <svg style={{
-      position: "absolute",
-      transform: "translate(-50%, -50%)"
-      }} xmlns="http://www.w3.org/2000/svg" className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-      <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={3} d="M15 19l-7-7 7-7" />
-    </svg>
-  )
-}
+	return (
+		<svg
+			style={{
+				position: "absolute",
+				transform: "translate(-50%, -50%)",
+			}}
+			xmlns="http://www.w3.org/2000/svg"
+			className="h-6 w-6"
+			fill="none"
+			viewBox="0 0 24 24"
+			stroke="currentColor"
+		>
+			<path
+				strokeLinecap="round"
+				strokeLinejoin="round"
+				strokeWidth={3}
+				d="M15 19l-7-7 7-7"
+			/>
+		</svg>
+	);
+};
 const ArrowRight = () => {
-  return (
-    <svg style={{
-      position: "absolute",
-      transform: "translate(-50%, -50%)"
-      }} xmlns="http://www.w3.org/2000/svg" className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-      <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={3} d="M9 5l7 7-7 7"  />
-    </svg>
-  )
-}
+	return (
+		<svg
+			style={{
+				position: "absolute",
+				transform: "translate(-50%, -50%)",
+			}}
+			xmlns="http://www.w3.org/2000/svg"
+			className="h-6 w-6"
+			fill="none"
+			viewBox="0 0 24 24"
+			stroke="currentColor"
+		>
+			<path
+				strokeLinecap="round"
+				strokeLinejoin="round"
+				strokeWidth={3}
+				d="M9 5l7 7-7 7"
+			/>
+		</svg>
+	);
+};
 const Prev = (props) => {
-  const { pathname } = useLocation()
-  const { className, style, onClick } = props;
-  return (
-    <div
-      className={`${className} arrow-custom-circle`}
-      style={{
-        ...style,
-        color: pathname !== '/' ? "#FFF" : "#11698E",
-        background: pathname !=='/' ? "no-repeat center #11698E" : "no-repeat center #FFF"
-        }}
-        onClick={onClick}
-        >
-        <ArrowLeft />
-    </div>
-  );
+	const { className, style, onClick, home } = props;
+	return (
+		<div
+			className={`${className} arrow-custom-circle`}
+			style={{
+				...style,
+				color: !home ? "#FFF" : "#11698E",
+				background: !home
+					? "no-repeat center #11698E"
+					: "no-repeat center #F8F1F1",
```

Comment:
Why do you use "not" condition? instead of `!home` you can to use `home` and do the code more easily of read. 

```suggestion
				color: home ? "#11698E" : "#FFF",
				background: home
					? "no-repeat center #F8F1F1"
					: "no-repeat center #11698E",
```